PointsRanking/Fun_PointsRanking.php

```
@@ -0,0 +1,593 @@
+        $row = safe_fetch($Rs);
+        $key = $row->PltcPresetKey;
+    }
+    safe_free_result($Rs);
+    return $key;
+}
+
+function pl_points_set_tournament_preset($tourId, $presetKey)
+{
+    $tourId = intval($tourId);
+
+    $Rs = safe_r_sql("SELECT PltcTournament FROM PLPointsTournamentConfig WHERE PltcTournament = " . $tourId);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $Rs. Configured minimum length is 3. (undefined)

(ShortVariable)